src/plugins/newsfeed/public/plugin.tsx

```diff
@@ -51,7 +51,10 @@ export class NewsfeedPublicPlugin
     return {
       createNewsFeed$: (endpoint: NewsfeedApiEndpoint) => {
         const config = Object.assign({}, this.config, {
-          service: { pathTemplate: `/${endpoint}/v{VERSION}.json` },
+          service: {
+            pathTemplate: `/${endpoint}/v{VERSION}.json`,
+            urlRoot: this.config.service.urlRoot,
+          },
```

Comment:
NIT: In case we add more options to the `service` attribute in the future, what do you think we spread the options just in case?
```suggestion
          service: {
            ...this.config.service,
            pathTemplate: `/${endpoint}/v{VERSION}.json`,
          },
```

Reply:
This makes sense to me, thanks!